src/geowombat/core/api.py

```diff
@@ -546,12 +546,12 @@ def __init__(
 
         self.data = self.data.assign_attrs(
             {
-                'data_are_separate': int(self.__data_are_separate),
-                'data_are_stacked': int(self.__data_are_stacked),
+                '_data_are_separate': int(self.__data_are_separate),
+                '_data_are_stacked': int(self.__data_are_stacked),

```

Comment:
Added underscore protection here because it conflicted with a property of the same name.